content/02.body.md

```diff
@@ -3,45 +3,45 @@
 This is especially true for networks encoding biomedical knowledge or assays, where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology to create non-uniform degree distributions (Figure {@fig:hetionet}).
 
 ![
 **Biomedical networks are characterized by non-uniform degree distributions.**
 Eight degree distributions are plotted for six edge types Hetionet v1.0 [@doi:10.7554/eLife.26726].
 Hetionet integrates subnetworks for 24 different edge types, the degree distributions of which are analyzed separately.
-Furthermore, bipartite (e.g. Anatomy→expresses→Gene) and directed (e.g. Gene→regulates→Gene) edge types have both source and target degrees that must be assessed separately.
+Furthermore, bipartite (e.g. Anatomy→expresses→Gene) and directed (e.g. Gene→regulates→Gene) graphs (Hetionet edge types) have both source and target degrees that must be assessed separately.
 Undirected edge types (e.g Compound–resembles–Compound) have only a single degree distribution.
 Degree distributions are non-uniform and vary greatly between different networks.
-The y-axis is log~10~-scaled to accomodate the common occurrence where most nodes have low degree while a small portion of nodes have high degree.
-Several distributions have node degree ceilings corresponding to a node being connected to all other possible nodes.
+The y-axis is log~10~-scaled to accommodate the common occurrence where most nodes have low degree while a small portion of nodes have high degree.
+Several distributions have nodes that reach the maximum degree, corresponding to a node being connected to all other possible nodes.
 Zero-degree nodes are not displayed, since methodological limitations often result in edge data only existing for a subset of nodes.
 ](https://github.com/greenelab/xswap-analysis/raw/b0db22c82b2e58bf1ef5ae78317167982016e26b/img/hetionet_degrees.png){#fig:hetionet width="100%"}
 
 Degree is an important metric for differentiating between nodes, and it appears in many common edge prediction features [@doi:10.1155/2015/172879].
 However, reliance on degree can pose problems for edge prediction.
-Firstly, bias in networks can distort node degree so that degree differences between two nodes may not be meaningful.
-Secondly, reliance on degree can lead edge prediction methods to make nonspecific or trivial predictions and fail to identify novel or insightful relationships.
+First, bias in networks can distort node degree so that a difference in degree between two nodes in a given network may not reflect a true difference in number of relationships.
+Second, edge prediction methods relying heavily on degree may be nonspecific---predicting trivial rather than insightful new relationships.
```

Comment:
```suggestion
Second, edge prediction methods that rely heavily on degree may be nonspecific---predicting trivial rather than insightful new relationships.
```